README.md

```diff
@@ -103,3 +103,9 @@ our [DingTalk group](https://qr.dingtalk.com/action/joingroup?code=v1,k1,GyDX5fU
 to engage with us.
 
 <img src="docs/dingtalk.png" height="200">
+
```

Comment:
Maybe an alternative is adding a `adopters/commerce/etc` badge instead of adding a section